scripts/scripts.js

```diff
@@ -16,7 +16,7 @@ import {
   waitForLCP,
 } from './lib-franklin.js';
 
-const LCP_BLOCKS = []; // add your LCP blocks to the list
+const LCP_BLOCKS = ['gallery']; // add your LCP blocks to the list
```

Comment:
do we add the carousel as well?

Reply:
actually thanks for spotting that, it was an experiment which made no difference at all. I'll remove it